test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -247,37 +270,49 @@ public Object invoke(Object proxy, Method method, Object[] args)
 
   @Test
   public void testBlobColumnModeServesResultBlob() throws Exception {
+    String content = "hello world";
+    executeUpdate("create table data(id int, content blob)");
+    executeUpdate("insert into data(id, content) "
+        + "values(1, file_read('" + writeTempFile(content) + "'))");
+
     MockResponse bar = new MockResponse();
     Response response = (Response) Proxy.newProxyInstance(
         Response.class.getClassLoader(),
         new Class[] { Response.class }, bar);
     Map<String, String> cfg = new TreeMap<String, String>();
-    cfg.put("columnName", "my-blob-col");
+    cfg.put("columnName", "content");
     ResponseGenerator resgen = ResponseGenerator.contentColumn(cfg);
-    String content = "hello world";
-    byte b[] = content.getBytes("US-ASCII");
-    resgen.generateResponse(
-        makeMockBlobResultSet(b, asList("my-blob-col"), asList(Types.BLOB)),
-        response);
-    String responseMsg = new String(bar.baos.toByteArray(), "US-ASCII");
-    Assert.assertEquals(content, responseMsg);
+
+    try (Statement stmt = getConnection().createStatement();
+        ResultSet rs = stmt.executeQuery("select * from data")) {
+      Assert.assertTrue("ResultSet is empty", rs.next());
+      resgen.generateResponse(rs, response);
+      String responseMsg = new String(bar.baos.toByteArray(), UTF_8);
+      Assert.assertEquals(content, responseMsg);
+    }
```

Comment:
I can't tell if you're suggesting that they should or shouldn't. We've always deleted the data in an `@After` method like I'm doing here, to reduce the complexity in the tests and make it impossible to forget.